serving-api/src/main/scala/com/stratio/sparkta/serving/api/service/http/PolicyHttpService.scala

```diff
@@ -345,12 +315,10 @@ trait PolicyHttpService extends BaseHttpService with SparktaSerializer {
           case ResponsePolicy(Success(policy)) => {
             val isValidAndMessageTuple = AggregationPoliciesValidator.validateDto(getPolicyWithFragments(policy))
             validate(isValidAndMessageTuple._1, isValidAndMessageTuple._2) {
-              complete {
-                val tempFile = File.createTempFile(s"${policy.id.get}-${policy.name}-", ".json")
-                respondWithHeader(`Content-Disposition`("attachment", Map("filename" -> s"${policy.name}.json"))) {
-                  scala.tools.nsc.io.File(tempFile).writeAll(write(policy))
-                  getFromFile(tempFile)
-                }
+              val tempFile = File.createTempFile(s"${policy.id.get}-${policy.name}-", ".json")
```

Comment:
Could be a good idea delete the temp file when the app finishes. It could be performed with the method:
```scala
tempFile.deleteOnExit()
```